src/v2/System/Router/Utils/findCurrentRoute.tsx

```diff
@@ -1,25 +1,17 @@
-import { Match } from "found"
-import { AppRouteConfig } from "../Route"
-
-export const findCurrentRoute = ({
-  route: baseRoute,
-  routes,
-  routeIndices,
-}: Match & { route?: AppRouteConfig }) => {
+export const findCurrentRoute = (
+  { route: baseRoute, routes, routeIndices }: any /* FIXME: Add type */
```

Comment:
I took the type off of here because sometimes all the finagling with `!` and `?` and so on just makes things even more unsafe. This file was a particularly bad offender. (I welcome another set of eyes.)

Reply:
What type does this wind up returning?

Reply:
It returns a [Route](https://github.com/4Catalyzer/found/blob/ac5b727d094588992743fbe9b3ef1343483f870f/types/index.d.ts#L213)